apps/core/Search.js

```diff
@@ -2,57 +2,87 @@
 
 import * as React from 'react';
 import { View, Button } from 'react-native';
```

Comment:
Please use button from universal-components instead

Reply:
Sure. I have a plan to swap it in next issue. Also with styling inputs according to final design and with implementation of trip and passengers selectors.